src/plugins/unified_search/public/dataview_picker/data_view_picker.tsx

```diff
@@ -18,7 +18,6 @@ export type ChangeDataViewTriggerProps = EuiButtonProps & {
 };
 
 export enum TextBasedLanguages {
-  SQL = 'SQL',
```

Comment:
I dropped this since we don't support SQL anymore and it was unused outside of tests.